nixos/doc/manual/release-notes/rl-1909.xml

```diff
@@ -547,8 +547,8 @@
       </para>
      </listitem>
     </itemizedlist>
-
-     This also configures the kernel to pass coredumps to <literal>systemd-coredump</literal>.
+     This also configures the kernel to pass coredumps to <literal>systemd-coredump</literal>,
+     and restricts the SysRq key combinations to the sync command only.
```

Comment:
by looking at https://www.kernel.org/doc/html/latest/admin-guide/sysrq.html I could see the other sysrg functions which weren't `enable sync command` so I knew which one's were disabled.